webui/react/src/pages/TrialDetails/Profiles/ProfilesEnabled.tsx

```diff
@@ -0,0 +1,56 @@
+import { Alert } from 'antd';
+import React, { useState } from 'react';
+
+import Section from 'components/Section';
+import { TrialDetails } from 'types';
+
+import SystemMetricChart from './SystemMetricChart';
+import SystemMetricFilter, { FiltersInterface } from './SystemMetricFilter';
+import TimingMetricChart from './TimingMetricChart';
+import { MetricType, useFetchMetrics } from './utils';
+
+export interface Props {
+  trial: TrialDetails;
+}
+
+const ProfilesEnabled: React.FC<Props> = ({ trial }: Props) => {
+  const [ filters, setFilters ] = useState<FiltersInterface>({});
+  const systemMetrics = useFetchMetrics(
+    trial.id,
+    MetricType.System,
+    filters.name,
+    filters.agentId,
+    filters.gpuUuid,
+  );
+
+  return (
+    <>
+
+      {systemMetrics.isEmpty && (
+        <Alert
+          message='No data available.'
+          type='warning'
+        />
+      )}
+
+      <div style={{ display: (systemMetrics.isEmpty ? 'none' : 'block') }}>
+
+        <Section
+          bodyBorder
+          filters={<SystemMetricFilter trial={trial} value={filters} onChange={setFilters} />}
+          title="System Metrics"
+        >
+          <SystemMetricChart systemMetrics={systemMetrics} />
+        </Section>
+
+        <Section bodyBorder title="Timing Metrics">
+          <TimingMetricChart trial={trial} />
+        </Section>
+
+      </div>
+
+    </>
+  );
```

Comment:
**suggestion:** use an early return for the `Alert`:
```
if (systemMetric.isEmpty) return <Alert ... />;

return (
  <Section ...>
    ...
  </Section>
);
```

Reply:
No, it's as is because <SystemMetricFilter> needs to render.